machine-learning/Dockerfile

```diff
@@ -1,19 +1,25 @@
-FROM python:3.10
+FROM python:3.10 as builder
 
-ENV TRANSFORMERS_CACHE=/cache \
-    PYTHONDONTWRITEBYTECODE=1 \
+ENV PYTHONDONTWRITEBYTECODE=1 \
     PYTHONUNBUFFERED=1 \
     PIP_NO_CACHE_DIR=true
 
-WORKDIR /usr/src/app
+RUN python -m venv /opt/venv && \
+    /opt/venv/bin/pip install --pre torch  -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html && \
+    /opt/venv/bin/pip install transformers tqdm numpy scikit-learn scipy nltk sentencepiece flask Pillow gunicorn && \
+    /opt/venv/bin/pip install --no-deps sentence-transformers
```

Comment:
Since this is a builder image, is there any advantage to combining these commands instead of having them separated?

Reply:
You're right, there's probably no need for this. Will update it to be separate commands :)